examples/internationalization/add_language/lib/main.dart

```diff
@@ -1,375 +1,52 @@
-  String get inputDateModeButtonLabel => 'Switch to input';
-
-  @override
-  String get inputTimeModeButtonLabel => 'Switch to text input mode';
-
-  @override
-  String get invalidDateFormatLabel => 'Invalid format.';
-
-  @override
-  String get invalidDateRangeLabel => 'Invalid range.';
-
-  @override
-  String get invalidTimeLabel => 'Enter a valid time';
-
-  @override
-  String get licensesPackageDetailTextOther => '\$licenseCount licenses';
-
-  @override
-  String get saveButtonLabel => 'SAVE';
-
-  @override
-  String get selectYearSemanticsLabel => 'Select year';
-
-  @override
-  String get timePickerDialHelpText => 'SELECT TIME';
-
-  @override
-  String get timePickerHourLabel => 'Hour';
-
-  @override
-  String get timePickerInputHelpText => 'ENTER TIME';
-
-  @override
-  String get timePickerMinuteLabel => 'Minute';
-
-  @override
-  String get unspecifiedDate => 'Date';
-
-  @override
-  String get unspecifiedDateRange => 'Date Range';
-}
-
-class Home extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      body: Center(
-        child: ElevatedButton(
-          onPressed: () {
-            // This would fail if localizations were not provided for the
-            // app's locale. Although only a few Material widgets display
-            // localized strings, many expect to be able to find localized
-            // values for tooltips and other strings related to accessibility.
-            showDialog(
-              context: context,
-              builder: (BuildContext context) =>
-                  Dialog(child: Center(child: Text('Hello World'))),
-            );
-          },
-          child: const Text('Show Dialog'),
-        ),
-      ),
-    );
-  }
-}
+import 'nn_intl.dart';
 
 void main() {
   runApp(
     MaterialApp(
       localizationsDelegates: [
         GlobalWidgetsLocalizations.delegate,
         GlobalMaterialLocalizations.delegate,
-        BeMaterialLocalizations.delegate,
+        NnMaterialLocalizations.delegate, // Add the newly created delegate
+      ],
+      supportedLocales: [
+        const Locale('en', 'US'),
+        const Locale('nn'),
       ],
-      supportedLocales: [const Locale('be', 'BY')],
       home: Home(),
     ),
   );
 }
+
+class Home extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Localizations.override(
+      context: context,
+      locale: const Locale('nn'),
+      child: Scaffold(
+        appBar: AppBar(),
+        drawer: Drawer(
+          child: ListTile(
+            leading: const Icon(Icons.change_history),
+            title: const Text('Change history'),
+            onTap: () {
+              Navigator.pop(context); // close the drawer
+            },
+          ),
+        ),
+        body: const Center(
+          child: Padding(
+            padding: EdgeInsets.all(50.0),
+            child: Text(
+              'Long press  hamburger icon in the app bar '
```

Comment:
Nit: Extra space between press and hamburger

Reply:
You might also want to clarify that we mean the drawer's menu icon aka the hamburger icon.